woocommerce-gateway-payfast.php

```diff
@@ -6,7 +6,7 @@
  * Description: Receive payments using the South African Payfast payments provider.
  * Author: WooCommerce
  * Author URI: https://woocommerce.com/
- * Version: 1.7.0
+ * Version: 1.7.1
```

Comment:
Version number will be bumped during the release process so this doesn't need updated
```suggestion
 * Version: 1.7.0
```